### PR TITLE
SDK-267 [Python SDK] File storage README

### PR DESCRIPTION
**Pull request by RommeroInCountry**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-267](https://incountry.atlassian.net/browse/SDK-267)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by igoralekseev — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -7,71 +7,69 @@ InCountry Storage SDK
...
 
-`client_id`, `client_secret`,  and `environment_id` can be fetched from your dashboard on `Incountry` site.
+The `client_id`, `client_secret`,  and `environment_id` variables can be fetched from your dashboard on `Incountry` Portal.
 
 
-`endpoint` defines API URL and is used to override default one.
+The `endpoint` variable defines API URL and is used to override the default one.
 
-You can turn off encryption (not recommended). Set `encrypt` property to `false` if you want to do this.
+You can disable the encryption (not recommended). Set the `encrypt` property to `false` if you want to disable it. Avoid using it when using for production data.
```
Comment (artemgolozubov) [inline README.md:52]:
```suggestion
You can disable the encryption (not recommended). Set the `encrypt` property to `false` if you want to disable it. Avoid disabling it when working with production data.
```

File: README.md [hunk 2]
```diff
@@ -497,43 +496,225 @@ delete_result = {
...
+
+
+### Adding attachments
+
+---
+Note:
+---
+
+The `add_attachment` method allows you to add or replace attachments.
+File data can be provided either as `BynaryIO` object or `string` with a path to the file in the file system.
```
Comment (artemgolozubov) [inline README.md:543]:
```suggestion
File data can be provided either as `BinaryIO` object or `string` with a path to the file in the file system.
```

**Review by artemgolozubov — COMMENTED** [review]
Fix a typo pls

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by igor-alekseev-dev — APPROVED** [review]

**Pull request merged**